firmware/Core/Inc/flash_bank/flash_bank.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __INCLUDE_GUARD__FLASH_BANK_H__
```

Comment:
Recommend renaming everything "flash bank"-related (files, functions, etc.) to "stm32_internal_flash" probably. The external flash (4x flash modules) is more of a "flash bank" than the internal flash imo; the name doesn't inherently differentiate.